api-reference/beta/api/profilephoto_update.md

```diff
@@ -13,9 +13,11 @@ Use only PUT for this operation in the beta version.
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
 
-- Profile photo of the signed-in **user** - User.ReadWrite, User.ReadWrite.All
-- Profile photo of a **group** - Group.ReadWrite.All
-- Photo of a **contact** - Contacts.ReadWrite
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account)     | Profile photo of the signed-in **user**:<br/>User.ReadWrite, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.ReadWrite |
+|Delegated (personal Microsoft account) | Profile photo of the signed-in **user**:<br/>User.ReadWrite, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.ReadWrite |
```

Comment:
Remove User.ReadWrite.All - this is not relevant in consumer.  However not sure this scenario even works.